cytosnake/utils/cyto_paths.py

```diff
@@ -0,0 +1,184 @@
+"""
+module: cyto_paths.py
+
+This module will contain functions that handles `cytosnake's` pathing
+"""
+
+from typing import Optional
+from pathlib import Path
+
+# cytosnake imports
+import cytosnake
+from cytosnake.utils.file_utils import file_search, find_project_dir
+from cytosnake.guards.path_guards import is_valid_path
+
+
+def get_meta_path() -> Path:
+    """returns meta path configurational file path.
+
+    Returns
+    -------
+    Path
+        Path object pointing to `_paths.yaml` config file
+    """
+
+    # getting project directory
+    proj_dir = find_project_dir()
+    if not proj_dir.exists():
+        raise FileNotFoundError("Current directory is not a project folder")
+
+    return proj_dir.absolute()
+
+
+# TODO: this should be placed in the guards module
```

Comment:
Is this PR out of scope to perform this TODO?

Reply:
Yes this is currently out of the scope. I let this TODO message for the `helper_functions` PR